lib/execution_engine2/utils/Condor.py

```diff
@@ -197,18 +197,14 @@ def run_job(self, params: JobSubmissionParameters) -> SubmissionInfo:
         TODO: Add a retry
         TODO: Add list of required params
         :param params:  Params to run the job.
-        :return:
+        :return: ClusterID, Submit File, and Info about Errors
         """
+        # Contains sensitive information to be sent to condor
         submit = self._create_submit(_not_falsy(params, "params"))
 
         sub = self.htcondor.Submit(submit)
         try:
             schedd = self.htcondor.Schedd()
-            self.logger.debug(schedd)
-            self.logger.debug(submit)
-            self.logger.debug(os.getuid())
-            self.logger.debug(pwd.getpwuid(os.getuid()).pw_name)
-            self.logger.debug(submit)
             with schedd.transaction() as txn:
                 return SubmissionInfo(str(sub.queue(txn, 1)), sub, None)
```

Comment:
Out of scope for this PR but I think the 2nd argument to `SubmissionInfo` is supposed to be `submit` vs. `sub`

Reply:
Although maybe it should be nothing so the secrets aren't exposed outside of the Condor module